test/adapter/json_api/links_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+
+LinkPost = Class.new(Model)
+class LinkPostSerializer < ActiveModel::Serializer
+  belongs_to :link_author
+end
+
+LinkAuthor = Class.new(Model)
+class LinkAuthorSerializer < ActiveModel::Serializer
+  links do
+    link :self, href: "//link_author/#{object.id}"
```

Comment:
How do you read 

- https://github.com/json-api/json-api/issues/840#issuecomment-141631828
- https://github.com/json-api/json-api/pull/834#issuecomment-131192713
- https://github.com/json-api/json-api/pull/834#issuecomment-141631788

>  relative references are simply not allowed.